### DISCUSSION TITLE
Callback after router navigation scroll to top

### DISCUSSION THREAD

**Discussion by thomasledoux1** [vercel/next.js#13267] [Help]
# Feature request

## Is your feature request related to a problem? Please describe.

I created a website where you have a Link component which goes to a new page (in /pages directory). 
By default the page scrolls to top, this is fine by me.
But I would like to trigger some animations _after_ the scrolling is finished.
I read the docs and could not find a way to do this?

## Describe the solution you'd like

It would help me if there was another event on the next/router for example: "routeChangeScrollComplete"

## Describe alternatives you've considered

Triggering the animations with the useEffect hook triggers them too early, and so does the routeChangeComplete event of the next/router...

## Additional context

You can see the behaviour by clicking the CV link on my website: https://thomasledoux.be.
The Github repo behind this website is: https://github.com/thomasledoux1/website-thomas

**Comment by jmoliugp** [comment]
Hey @thomasledoux1 ,

Thanks for bringing this up! I'm also interested in using this feature. Any updates on whether this has been implemented or if there's a workaround available?
